packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/index.tsx

```diff
@@ -38,11 +38,25 @@ import {
 } from 'store/settings/advanced'
 import AnalyticsService from 'services/analytics/AnalyticsService'
 import { ScrollView } from 'react-native-gesture-handler'
+import { Platform } from 'react-native'
+import useInAppBrowser from 'common/hooks/useInAppBrowser'
+import { selectContacts } from 'store/addressBook'
+import Logger from 'utils/Logger'
+import {
+  HELP_URL,
+  PRIVACY_POLICY_URL,
+  TERMS_OF_USE_URL
+} from 'resources/Constants'
+import DeviceInfo from 'react-native-device-info'
+import { Space } from 'components/Space'
+
+const PRESELECTED_PLATFORM =
+  Platform.OS === 'ios' ? 'Core+mobile+(iOS)' : 'Core+mobile+(Android)'
 
 const AccountSettingsScreen = (): JSX.Element => {
   const { deleteWallet } = useDeleteWallet()
   const dispatch = useDispatch()
-  const isDeveloperMode = useSelector(selectIsDeveloperMode)
+  const isDeveloperModeEnabled = useSelector(selectIsDeveloperMode)
```

Comment:
if you want to you can create selectIsDeveloperModeEnabled and add a @deprecated comment to the other one